### PR TITLE
PubSub 101 Creating Data Flows With Topics

### PR DESCRIPTION
**Pull request by pdeziel**

### CODE REVIEW

File: content/en/blog/2023-11-17-pubsub-101---using-the-python-sdk.md [hunk 1]
```diff
@@ -76,37 +76,7 @@ Once you've created a client, make sure you have a topic to publish to. The `get
...
-created {
-  seconds: 1694443634
-  nanos: 72367968
-}
-modified {
-  seconds: 1695659671
-  nanos: 740859516
-}
-]
+[Topic(id=01HFT1FK687DSK7YSF3Q6WQYGT, name=my-topic, ), Topic(id=01HFT1FTNMAAD1B9DG8AB9V1ZD, name=my-other-topic, )]
```
Comment (pdeziel) [inline content/en/blog/2023-11-17-pubsub-101---using-the-python-sdk.md:79]: Fixing the previous blog post so this matches what is actually printed out.

File: static/img/blog/waterfall.jpg
Comment (pdeziel) [inline static/img/blog/waterfall.jpg:1]: Open to suggestions on a better image so this is a placeholder for now.

File: content/en/blog/2023-11-20-pubsub-101---creating-data-flows-with-topics.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## How to think about topics
+
+An Ensign `topic` is a globally ordered and globally distributed stream of `events`. They are somewhat analagous to the concept of tables in a relational database, however there are a few important differences:
+
+Property | Tables                                  |  Topics
+---------|-----------------------------------------|-----------------------------------------------
+Mutability | Can create, read, update, delete rows | Events are immutable - can only publish new Events
+Heterogenity | Columns usually have singular data types  | Events can have varying data types and schemas
+Ordering | Rows are not explicitly ordered and ordering usually happens at query time | Events have a defined ordering and the ordering is consistent across replicas
```
Comment (pdeziel) [inline content/en/blog/2023-11-20-pubsub-101---creating-data-flows-with-topics.md:34]: Any other differences worth mentioning here?

**Review by rebeccabilbro — APPROVED** [review]
Looks great @pdeziel ! I really like how you're breaking these lessons down into small and easily consumable modules!

**Pull request merged**